README.md

```diff
@@ -37,6 +37,7 @@ module.exports = {
     new FileIncludeWebpackPlugin(
       {
         source: './src/templates', // relative path to your templates
+        outputPath: '../html',
```

Comment:
Add this as another section in read me under the heading `How to change the output destination?`.
Do not change the default configuration readme.
change `outputPath` to `destination` and add the following statement.

> `destination` is an optional configuration which defaults to `output.path` in webpack config.